node_modules/gh-core/lib/db.js

```diff
@@ -180,9 +180,11 @@ var _setUpModel = function(sequelize) {
                 return true;
             },
             'toJSON': function() {
+                var json = _.clone(this.dataValues);
```

Comment:
There are places outside of REST serialisation now where we do `instance.toJSON()`. Serialising an instance shouldn't have any side-effects such as properties being gone.